src/interp/interp.h

```diff
@@ -414,6 +414,8 @@ class FreeList {
   Index New(Args&&...);
   void Delete(Index);
 
+  FreeList& operator=(FreeList&&);
```

Comment:
That about the copy assignment operator?   Is that no longer implicitly defined?

i.e. what happens if you do the following:

```  
wabt::interp::Store store1;
wabt::interp::Store store2;
store1 = store2;
```

Does that now fail to compile?  Or do you have to explictly add:

```
operator= (const FreeList&) = delete;
```

?

Reply:
It sounds like it should not be needed since:

```
Deleted implicitly-declared copy assignment operator
An implicitly-declared copy assignment operator for class T is defined as deleted if any of the following is true:

T has a user-declared move constructor;
T has a user-declared move assignment operator.
```